### DISCUSSION TITLE
Accident after I installed k3s

### DISCUSSION THREAD

**Discussion by cubxxw** [k3s-io/k3s#6563] [Q&A]
Ladies and gentlemen, after the agent node deployment, my k3s prompts such an error. What is the cause

![image](https://user-images.githubusercontent.com/86140903/204072112-01770e1f-feb3-4290-97ff-b573294be580.png)

server：
```bash
INSTALL_K3S_MIRROR=cn K3S_NODE_NAME=cubmaster01     K3S_KUBECONFIG_OUTPUT=/home/escape/.kube/config     INSTALL_K3S_EXEC="--docker" | sh install.sh 
```

agent：
```bash
INSTALL_K3S_MIRROR=cn K3S_NODE_NAME=cubnode01     K3S_KUBECONFIG_OUTPUT=/home/escape/.kube/config     K3S_URL=https://192.168.71.130:6443/     K3S_TOKEN=K100....  sh install.sh
```

**Comment by brandond** [comment]
If `kubectl` is trying to use `localhost:8080` as the server address, then you don't have a kubeconfig available. Are you running this command on the server, or did you copy the file over from the server, and set the KUBECONFIG environment variable to the path where you placed the file?

Note that agents don't have a copy of the admin kubeconfig, only servers. If you're trying to copy the admin kubeconfig off the agent, that is not going to work.
